platform/spire/src/access.py

```diff
@@ -42,7 +42,7 @@ def needs_rotate(path, interval=DEFAULT_ROTATE_INTERVAL):
 
```

Comment:
This commit message overflows the maximum allowable length of the first line, according to GitHub.

Reply:
github comments are associated with files, not commits.  but I will go back and check all of them

Reply:
oh no woops. I'm talking about d657905337588ed7dbbbdaeb983eb1d1fa08aee4.